test/e2e/kubelet.go

```diff
@@ -420,28 +451,21 @@ var _ = framework.KubeDescribe("kubelet", func() {
 			})
 
 			// execute It blocks from above table of tests
-			for _, test := range testTbl {
-				t := test // local copy for closure
+			for _, t := range testTbl {
 				It(fmt.Sprintf("%v [Serial]", t.itDescr), func() {
```

Comment:
If they are both run serially, it's probably better to move up the `[Serial]` tag for visibility.  Haven't checked but I'm pretty sure the test output will read something like:
```
host cleanup with volume mounts [Jeff][HostCleanup][Slow]
        Host cleanup after pod using NFS mount is deleted [Volume][NFS]
                after deleting the nfs-server, the host should be cleaned-up when deleting sleeping pod which mounts an NFS vol  [Serial]
```